com.unity.netcode.gameobjects/Tests/Editor/com.unity.netcode.editortests.asmdef

```diff
@@ -5,19 +5,27 @@
         "Unity.Netcode.Runtime",
         "Unity.Netcode.Editor",
         "Unity.Multiplayer.MetricTypes",
-        "Unity.Multiplayer.NetStats"
+        "Unity.Multiplayer.NetStats",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+    "includePlatforms": [],
```

Comment:
@MFatihMAR I'm waiting for you to comment on these :P

Reply:
I reverted this to the develop branch as it is not needed for this PR scope.